src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclets/toolkit/taglets/SnippetTaglet.java

```diff
@@ -215,14 +214,14 @@ private Content generateContent(Element holder, DocTree tag, TagletWriter writer
             // we didn't create JavaFileManager, so we won't close it; even if an error occurs
             var fileManager = writer.configuration().getFileManager();
 
-            // first, look in local snippet-files subdirectory
-            Utils utils = writer.configuration().utils;
-            PackageElement pkg = getPackageElement(holder, utils);
-            JavaFileManager.Location l = utils.getLocationForPackage(pkg);
-            String relativeName = "snippet-files/" + v;
-            String packageName = packageName(pkg, utils);
```

Comment:
Drop the now-unused `packageName` method.

Reply:
Good catch!  Done.